src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -89,20 +98,38 @@ export class PassFinalisationPage extends BasePageComponent {
         select(getPassCompletion),
         select(getPassCertificateNumber),
       ),
+      provisionalLicenseValue: null,
     };
     this.inputSubscriptions = [
       this.inputChangeSubscriptionDispatchingAction(this.passCertificateNumberInput, PassCertificateNumberChanged),
     ];
+    this.initialStateSubscriptions = [
+      this.provisionalLicenseSubscription(),
+    ];
   }
 
   ngOnDestroy(): void {
     this.inputSubscriptions.forEach(sub => sub.unsubscribe());
+    this.initialStateSubscriptions.forEach(sub => sub.unsubscribe());
   }
 
   ionViewDidEnter(): void {
     this.store$.dispatch(new PassFinalisationViewDidEnter());
   }
 
+  provisionalLicenseReceived(): void {
+    this.store$.dispatch(new ProvisionalLicenseReceived());
+  }
+
+  provisionalLicenseNotReceived(): void {
+    this.store$.dispatch(new ProvisionalLicenseNotReceived());
+  }
+
+  setProvisionalLicenseValue = (provisionalLicenseValue: boolean): void => {
```

Comment:
I don't think you need this function + complications around subscription if you subscribe directly to the store as opposed to keeping a copy of the state in the component.